Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -190,13 +275,13 @@ public IAsyncLoop Mine(WalletSecret walletSecret)
 
             this.rpcGetStakingInfoModel.Enabled = true;
 
-            this.mining = this.asyncLoopFactory.Run("PosMining.Mine", token =>
+            this.mining = this.asyncLoopFactory.Run("PosMining.Mine", async token =>
             {
                 this.logger.LogTrace("()");
 
                 try
                 {
-                    this.GenerateBlocks(walletSecret);
+                    await this.GenerateBlocks(walletSecret);
```

Comment:
We decided on to ConfigureAwait(false) in libraries (Stratis.Bitcoin, NStratis and Features) and not ConfigureAwait(false) in in Tests, Api, Cli and Daemon projects.  
https://github.com/stratisproject/StratisBitcoinFullNode/pull/568#issuecomment-336212110

Reply:
k, tx, done